openshift/sandboxed-containers-operator
# Remove unused code

**- Description of the problem which is fixed/What is the use case**
Remove unused code
Fixes: #121 

**- What I did**
Removed the code files and objects that are not in use.

**- How to verify it**
Deploy the operator either via CLI or UI and you shouldn't see the KataInstallConfig object in the KataConfig CRD

**- Description for the changelog**

Remove unused code and object
